src/registrar/templates/includes/portfolio_request_review_steps.html

```diff
@@ -8,15 +8,24 @@
     {% endif %}
 
     {% if step == Step.REQUESTING_ENTITY %}
-        {% if domain_request.organization_name %}
-            {% with title=form_titles|get_item:step value=domain_request %}
-                {% include "includes/summary_item.html" with title=title value=value heading_level=heading_level editable=is_editable edit_link=domain_request_url address='true' %}
-            {% endwith %}
+        {% with title=form_titles|get_item:step %}
+        {% if domain_request.sub_organization %}
+            {% include "includes/summary_item.html" with value=domain_request.sub_organization edit_link=domain_request_url %}
```

Comment:
Q:
Selecting top level organization shows its city/state on the summary page. Same for selection a other suborg, it shows the custom suborg city state. Selecting an existing suborg is the only scenario that does not show a city/state. 
 
![Screenshot 2024-11-01 at 1 43 41 AM](https://github.com/user-attachments/assets/12da37da-5d83-49d0-b277-79e7116a430e)

Reply:
City and state definitely shouldn't show if the user selected another (real) suborg.